kmeans-sklearn/README.md

```diff
@@ -0,0 +1,43 @@
+# KMeans via sklearn (Python2)
+
+Two clustering examples using [sklearn](https://scikit-learn.org/stable/)
+
+It is assumed you already have a Binaris account. If you don't have an account yet, worry not, signing up is painless and takes just 2 minutes. Visit [Getting Started](https://dev.binaris.com/tutorials/python/getting-started/) to find concise instructions for the process.
```

Comment:
Maybe something shorter:

If this is your first time using Binaris, visit our [Getting Started] page to set up your account for free.

Reply:
In any case, "it is assumed" is academic and a bit stilted.  I'd prefer e.g. "You should already have a Binaris account".

Reply:
Do you like the new version?